### ISSUE TITLE
Bad value for type long : AB

### ISSUE THREAD

**Issue by ebautista** [speedment/speedment#611]
I have a table with primary key that is defined as String, but the AbstractDbmsOperationHandler seems to retrieve PK as a generatedKey (even if it is not a generatedd key) and being converted to Long. 

Attached speedment.json, the table in question is "Area".

[speedment.zip](https://github.com/speedment/speedment/files/1896846/speedment.zip)

Using:
<speedment.version>3.0.9</speedment.version>
<db.groupId>org.postgresql</db.groupId>
<db.artifactId>postgresql</db.artifactId>
<db.version>42.1.4</db.version>

**Comment by ebautista** [comment]
I have done a workaround on the issue by adding a SERIAL "id" as the first column which guarantees that sqlStatement.addGeneratedKey(generatedKeys.getLong(1)); will always get a numeric column . This is highly inconvenient since it changes my table structure.

**Comment by lawesson** [comment]
Issue #293 is solved in Speedment 3.0.10. I have verified that the following table works for 3.0.10 but not 3.0.9.

```sql
CREATE TABLE stringid
(
    id character varying(255) NOT NULL,
    email character varying(255),
    CONSTRAINT primarykey PRIMARY KEY (id)
);
```

Does updating the pom.xml to speedment.version 3.0.10 solve also this issue?

**Comment by lawesson** [comment]
Closing issue, assuming it is solved in 3.0.10.
